front/src/components/list-of-group/GroupListPage.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { Layout, Menu } from 'antd';
+import 'antd/dist/antd.css';
+import { AddGroupButton, SiderStyle } from './styledListOfGroup';
+import { setCurrentGroup } from '../../common/redux/actions/groupListAction';
+import { connect } from 'react-redux';
+
+const { Content } = Layout;
+
+class GroupListPage extends Component {
+    onHandleGroup = (data) => {
```

Comment:
On `componentDidMount` you should send a request and get all of the groups that is available for logged user

Reply:
done